### ISSUE TITLE
Disable-other-villager: false causing player shops to no work

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#26]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/16

**Originally posted by Ythmevge (Aug 19, 2012):** 

What steps will reproduce the problem?
1. Setting disable-other-villagers to false
2. Setup a Play shop, using a spawn egg, right clicking on a chest, then the block you want him on. After setting an item or for sale it wouldn't show up for sale. Then when you try to shift right click on the villager to change his sale nothing will happen.
3. Then I changed disable-other-villager to true and after recreating the player shop it worked again.What version of the product are you using?
v 1.4 of ShopkeeperPlease provide any additional information below.
The server is running the following plugins:
2012-08-18 17:15:11 [INFO] Plugins (17): WorldBorder, CreeperHeal, WeatherSync, Minequery, NoCheatPlus, Shopkeepers, WorldPos, SkylandsPlus, BKCommonLib, UltraHardcore, Multiverse-Core, PermissionsEx, Modifyworld, CHDistantFarm, Multiverse-Inventories, Multiverse-SignPortals, Multiverse-NetherPortals

**Comment by blablubbabcDEV** [comment]
**Originally commented by nisovin (Aug 19, 2012):** 

Unable to reproduce. I think you made one test, and assumed that the change in the config resulted in the plugin working, but that is highly unlikely. I think something else probably happened.
You can see exactly what is happening when you right-click a villager if you set the debug option to true in the config.

            Edited Aug 19, 2012

**Comment by blablubbabcDEV** [comment]
**Originally commented by Ythmevge (Aug 19, 2012):** 

I think I know what I did, me and a friend tested multiple times, up to the point that I just deleted the save and config to get back to default. At that point it worked. Not sure why I thought the problem had to do with the other-villager option. Its all working now.
I backtracked my steps to try and find the cause, and it had to do with me setting the High currency value to 0. My though was that 0 would disable it and make it independent of currency item. Instead it causes a division by 0 error. and the villager Editor is inaccessible. 
Sorry for wasting your time, should have noticed the debug option.
